docs/installation/prepare/rpi-setup.md

```diff
@@ -171,27 +171,33 @@ sudo apt upgrade
 
 ## Enable Communication Protocols
 
-If you are using hardware that requires I2C, SPI, serial, or one-wire protocols to communicate with your Pi, you will need to enable them using `raspi-config`.
+Certain hardware, such as analog-to-digital converters (ADCs), accelerometers, and sensors, communicate with your Pi using specialized communications protocols, including I2C, SPI, serial, or one-wire protocols.
```

Comment:
[nit] "Certain hardware" is singular but then the list is of plural items so "communicate" works for the plural but not the singular.